protocol/x/vault/keeper/orders_test.go

```diff
@@ -1 +1,424 @@
 package keeper_test
+
+import (
+	"math"
+	"testing"
+
+	"github.com/cometbft/cometbft/types"
+	"github.com/dydxprotocol/v4-chain/protocol/lib"
+	testapp "github.com/dydxprotocol/v4-chain/protocol/testutil/app"
+	"github.com/dydxprotocol/v4-chain/protocol/testutil/constants"
+	clobtypes "github.com/dydxprotocol/v4-chain/protocol/x/clob/types"
+	perptypes "github.com/dydxprotocol/v4-chain/protocol/x/perpetuals/types"
+	pricestypes "github.com/dydxprotocol/v4-chain/protocol/x/prices/types"
+	satypes "github.com/dydxprotocol/v4-chain/protocol/x/subaccounts/types"
+	vaulttypes "github.com/dydxprotocol/v4-chain/protocol/x/vault/types"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetVaultClobOrders(t *testing.T) {
+	// Set up test clob pairs, markets, perpetuals, and vault params.
+	testClobPairs := []clobtypes.ClobPair{
+		constants.ClobPair_Btc,
+		constants.ClobPair_Eth,
+	}
+	testMarketParams := []pricestypes.MarketParam{
+		constants.TestMarketParams[0],
+		{
+			Id:                 constants.TestMarketParams[1].Id,
+			Pair:               constants.TestMarketParams[1].Pair,
+			Exponent:           constants.TestMarketParams[1].Exponent,
+			MinExchanges:       constants.TestMarketParams[1].MinExchanges,
+			MinPriceChangePpm:  4_200, // Set a high min price change to test spread calculation.
+			ExchangeConfigJson: constants.TestMarketParams[1].ExchangeConfigJson,
+		},
+	}
+	testMarketPrices := []pricestypes.MarketPrice{
+		constants.TestMarketPrices[0],
+		constants.TestMarketPrices[1],
+	}
+	testPerps := []perptypes.Perpetual{
+		constants.BtcUsd_0DefaultFunding_0AtomicResolution,
+		constants.EthUsd_0DefaultFunding_9AtomicResolution,
+	}
+	// TODO (TRA-118): store vault strategy constants in x/vault state.
+	minBaseSpreadPpm := uint32(3_000)                   // 30bps
+	baseSpreadMinPriceChangePremiumPpm := uint32(1_500) // 15bps
+	orderExpirationSeconds := uint32(5)                 // 5 seconds
```

Comment:
nit: Use the constants declared in `x/vault`.

Reply:
In a later task, I'm going to remove the constants in `x/vault` and in this test file, initialize `x/vault` with these test params. will probably keep these test constants here (otherwise have to add them back in)